pgml-extension/src/lib.rs

```diff
@@ -21,9 +21,20 @@ pg_module_magic!();
 
 extension_sql_file!("../sql/schema.sql", name = "schema");
 
+extern "C" {
+    fn omp_set_num_threads(num_threads: i32);
+}
+
 #[cfg(not(feature = "use_as_lib"))]
 #[pg_guard]
 pub extern "C" fn _PG_init() {
+    config::initialize_server_params();
+    let omp_num_threads = config::PGML_OMP_NUM_THREADS.1.get();
+    if omp_num_threads > 0 {
```

Comment:
We can sanitize the value (ensure positive) in the `Lazy` initialization function, I think.

Reply:
Actually, I'm doing it deliberately. When the user specifies it to be `0`, we will not limit the number of threads that OpenMP spawns.